tox.ini

```diff
@@ -26,6 +26,7 @@ deps=
 	pytest
 	mock
 	Mako
+	decorator
```

Comment:
wait, you want this in requirements.txt, not here, right?

Reply:
i want it somewhere other than here for sure